docs/DeveloperGuide.md

```diff
@@ -6,33 +6,360 @@
+ Owner Name: Abel | Contact Number: 92929292
+ Vaccination added successfully!
+______________________________________________________________________
+```
+
+5. Test case: `vaccination at/Dog an/Russ on/Abel cn/92929292 v/Covid vd/12-12-2023 vt/19:00`
+   Expected output: A vaccination appointment is not added due to invalid date description.
+   Example: 
+```
+______________________________________________________________________
+ Sorry, an error was encountered! Here is the error description:
+ The date/time description is invalid.
+______________________________________________________________________
+```
+
+6. Test case: `surgery at/Hamster an/Polly on/Sam cn/93939393 sd/2023-12-12 st/19:00 ed/2023-12-12 et/20:00 p/H`
+   Expected output: A surgery appointment has been added successfully, details shown in the status message.
+   Example:
+```
+______________________________________________________________________
+ This appointment has been added to your appointment manager: 
+ UID: S14837641R | Priority: HIGH
+ Animal Name: Polly | Animal Type: Hamster
+ Owner Name: Sam | Contact Number: 93939393
+ Start Date: 2023-12-12 | Start Time: 19:00
+ End Date: 2023-12-12 | End Time: 20:00
+ Surgery added successfully!
+______________________________________________________________________
+```
+
+7. Test case: `surgery at/Hamster an/Polly on/Sam cn/93939393 sd/2023-12-12 st/19:00 ed/2023-12-12 et/19:00 p/H`
+   Expected output: A surgery appointment is not added due to dates and times being the same .
+   Example:
+```
+______________________________________________________________________
+ Sorry, an error was encountered! Here is the error description:
+ The start time cannot be the same as the end time!
+______________________________________________________________________
+```
+
+#### Listing appointments
+
+1. Prerequisites: Add an appointment using any of the add commands. At least one appointment in the list.
+2. Test case: `list`<br>
+   Expected output: A list of all previously added appointments.
+   Example:
+```
+______________________________________________________________________
+1.  UID: C84120821A
+ Date: 2023-12-12 | Time: 19:00
+ Animal Name: Lulu | Animal Type: Cat
+ Owner Name: Jon | Contact Number: 91919191
+2.  UID: V12400172X | vaccine: Covid
+ Date: 2023-12-12 | Time: 19:00
+ Animal Name: Russ | Animal Type: Dog
+ Owner Name: Abel | Contact Number: 92929292
+3.  UID: S14837641R | Priority: HIGH
+ Animal Name: Polly | Animal Type: Hamster
+ Owner Name: Sam | Contact Number: 93939393
+ Start Date: 2023-12-12 | Start Time: 19:00
+ End Date: 2023-12-12 | End Time: 20:00
+______________________________________________________________________
+```
+
+#### Finding appointments
+
+1. Prerequisites: Add an appointment using any of the add commands. At least one appointment in the list.
+2. Test case: `find a/Dog`
+   Expected output: A list of all previously added appointments with the animal type.
+   Example:
+```
```

Comment:
![image](https://user-images.githubusercontent.com/88373092/228761785-1b255e27-2dcd-4ae9-a8ce-1c3911a9f05a.png)

Within the DG page, there seems to be incorrectly formatted markdown code.
Consider adding newlines to try to remedy the errors.